docs/AccountType.md

```diff
@@ -0,0 +1,15 @@
+# Freeclimb::AccountType
+
+## Properties
+
+| Name | Type | Description | Notes |
+| ---- | ---- | ----------- | ----- |
+
```

Comment:
This should contain the enum property values. Looking at the rest of the enum based docs this seems to be a consistent error